src/client/components/EditUserForm/EditUserForm.tsx

```diff
@@ -72,6 +73,7 @@ const EditUserForm: React.FC<{ user: User }> = ({ user }) => {
     <div className="edit-user__form-container">
       <ProfilePicture userId={user.id} onChange={(profilePicture: File) => setProfilePicture(profilePicture)} />
       <TextInputFields user={user} onChange={changeUser} />
+      {user.roles[0].role === RoleName.COLLABORATOR && <CollaboratorPermissions user={user} />}
```

Comment:
i have already noticed that it's using roles[0] to identify role for country.
however this is incorrect,
1. user should have fetched all roles, not only current country role (edit user form can be access from admin page) (country role should be accessed using some utility Users.getCountryRole({user,countryIso})
2. here you need to check that countryIso exists.

do you want to fix this user.roles[0] inconsistency in this PR or a separate issue ?

Reply:
this is always true when we get users using getCountryUsers but you're right, not in that case. we can replace all the user.roles[0] with Users.getCountryRole. I can do in this pr.

<img width="565" alt="Schermata 2022-09-30 alle 10 00 35" src="https://user-images.githubusercontent.com/10047945/193222059-aff44853-171f-475f-8b5a-c8424bbcc067.png">

Reply:
It's also about fetching all roles associated to each user . That is the source of the inconsistency